src/styles/index.scss

```diff
@@ -1,3 +1,52 @@
+@import '/src/styles/variables';
+
 body {
+  @include center-all;
+
   margin: 0;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc((7 * $field-side) + (6 * $calendar-gap));
+  gap: $calendar-gap;
+
+  @each $start-day, $count in $days {
+    &--start-day-#{$start-day} :first-child {
+      margin-left: calc(($field-side + $calendar-gap) * $count);
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length-#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+}
+
+.calendar__field {
+  @include center-all;
+
+  height: $field-side;
+  width: $field-side;
+  box-sizing: border-box;
+  background-color: $main-color;
+  border: 1px solid black;
+  font-size: 30px;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
The `font-family` property is usually set in the root element (in this case, the `body` element) to make it inherited by all the children elements. It's better to move it to the `body` class.